check_stability.py

```diff
@@ -33,6 +33,7 @@
 wptrunner_root = None
 
 logger = logging.getLogger(os.path.splitext(__file__)[0])
+MAX_TABLE_ROWS = 256 if "CONTINUOUS_INTEGRATION" in os.environ else float("inf")
```

Comment:
Python appears to interpret `None` as being "less than" all other number values:

    >>> [(None < x) for x in [float('inf'), 1, 0, -1, -float('inf')]]
    [True, True, True, True, True]

Since we don't need to enforce a log size limit outside of continuous integration, I don't think this appropriate for use as a maximum value.